packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.Props.ts

```diff
@@ -176,141 +172,4 @@ export interface IContextualMenuProps extends React.Props<ContextualMenu> {
    */
   calloutProps?: ICalloutProps;
 
-}
```

Comment:
no i'm talking about the documentation page. You just removed the documentation by moving it to another file.